www/www-static.yml

```diff
@@ -29,6 +18,19 @@
     when: not _phpbbforum_style_file_st.stat.exists
 
   tasks:
+  - name: install deployment script
+    become: yes
+    template:
+      src: files/deploy
+      dest: /usr/local/bin/deploy
+      mode: 0555
+
+  - name: run deployment script
+    become: yes
+    command: /usr/local/bin/deploy -f
+    register: deploy_result
+    changed_when: '"Symlinked" in deploy_result.stdout'
```

Comment:
Probably fine, but if you want to make this a but more robust you could make the script set an exit code to indicate the status (optionally with a flag to activate it if you want to keep the default exit code of `0` for manual runs)

Reply:
Agreed. That's what I tried first but the non-zero exit code failed my command - see https://github.com/sbesson/prod-playbooks/runs/1557233100?check_suite_focus=true.

I have been contemplating whether there was a better way to communicate a changed status. Thoughts?

Reply:
You can use `failed_when` and check the return code: https://docs.ansible.com/ansible/latest/user_guide/playbooks_error_handling.html#defining-failure

Reply:
Thanks 6ee884e now implements a behavior similar to `diff` (or `git diff`) with `0 == successful, no change` , `1 == successful, changes` and anything above 1 meaning an error